src/index.tsx

```diff
@@ -4,11 +4,13 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { Provider } from 'react-redux';
 import { BrowserRouter } from 'react-router-dom';
-import { store } from 'redux/redux-store';
+
+import { setupStore } from 'store/store';
 
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
+const store = setupStore();
```

Comment:
why do use this setuoStore wrapper func ? why not just import it like it was before